### DISCUSSION TITLE
Best Practices for Code Reviews

### DISCUSSION THREAD

**Discussion by vatscodes1** [EddieHubCommunity/support#9112] [First-Timers] [answered]
As our team grows, it's becoming more important to have a consistent and effective code review process. Any advice?

**Comment by dinxsh** [comment, marked as answer]
agree on coding standards, use tools for linting and testing, focus on the code, not the coder, regular, quick reviews work best, and pair programming can help.

**Reply by uttam-rana** [reply]
Agreed as all of us have different styles of writing code we can agree on the certain coding standards that needs to be followed by every developer in the organisation no questions asked, also tools like ESLisnt and others are also useful.
